docs/usage/break-lock.rst.inc

```diff
@@ -29,7 +29,7 @@ borg break-lock
       --no-files-cache      do not load/update the file metadata cache used to
                             detect unchanged files
       --umask M             set umask to M (local and remote, default: 0077)
-      --remote-path PATH    set remote path to executable (default: "borg")
+      --remote-path PATH    use PATH as borg-executable on the remote (default: "borg")
```

Comment:
maybe rather don't recreate the usage files, we do that at release time to have less conflicts.